docs/taqo_install_instructions.md

```diff
@@ -0,0 +1,137 @@
+# Paco:Taqo Linux Installation Instructions
+
+**Table of Contents**
+
+
+[TOC]
+
+### System Requirements {#system-requirements}
+
+**OS:** Linux (64bit)
+
+**OS Version:** Any Google/glinux distribution should work. gLinux laptops and Cloudtop should work as well but we have not tested these.
```

Comment:
I actually did my tests on a Cloudtop. I think we can count it as a tested platform.

Reply:
Updated